tests/adapter/dbt/tests/adapter/python_model/test_spark.py

```diff
@@ -57,6 +57,25 @@ def model(dbt, session):
     return df
 """
 
+KOALAS_MODEL = """
+import databricks.koalas as ks
```

Comment:
I think that Koalas DataFrames can also be used on BigQuery even though the package is published by Databricks.

I did manual testing of Koalas on BigQuery here:
https://github.com/dbt-labs/dbt-bigquery/pull/321

Reply:
Koalas is a separate package from Spark and should be installed in the target cluster to pass this test.

Reply:
Thank you for this info @ueshin 👍

Do you have any instructions we could follow for installing `databricks.koalas` on the cluster we use for our CI for dbt-spark (via CircleCI)?

I have [this](https://github.com/dbt-labs/dbt-spark/pull/474) PR open for dbt-spark which you will automatically inherit for dbt-databricks.

Reply:
When the target cluster is DBR `>=7.1,<11.0`, Koalas should be available as it's pre-installed; otherwise, you need to install it in the cluster UI.

![Screen Shot 2022-09-26 at 13 35 42](https://user-images.githubusercontent.com/506656/192374945-e7fa6968-829c-4c17-87e4-76543a68c40a.png)